### ISSUE TITLE
Any way to override keyboard navigation handlers?

### ISSUE THREAD

**Issue by jakebanks** [elrumordelaluz/reactour#531]
Hi,

First off, thanks for the fantastic package. 

Is there any way to override the keyboard navigation handlers, in a similar way to being able to give an `onClick` handler to the next and previous buttons?

The use case is: my next button's click handler is responsible for determining the next step of the tour by skipping over `n` elements that don't appear on the page at all. I couldn't find a way to achieve this and support keyboard navigation.

Many thanks,
Jake

**Comment by elrumordelaluz** [comment]
Hi @jakebanks, thanks for open the _Issue_ and sorry for the late reply.
The new prop [keyboardHandler](https://github.com/elrumordelaluz/reactour/tree/main/packages/tour#keyboardhandler-keyboardhandler) should be now available in `@reactour/tour@3.3.0`.

You could use it this way:

```js
<TourProvider
  steps={steps}
  disableInteraction
  keyboardHandler={(e, clickProps) => {
    if (e.key === 'ArrowRight') {
      clickProps.setCurrentStep(
        Math.min(clickProps.currentStep + 1, clickProps.steps.length - 1)
      )
    }
    if (e.key === 'ArrowLeft') {
      clickProps.setCurrentStep(Math.max(clickProps.currentStep - 1, 0))
    }
    if (e.key === 'Escape') {
      const nextStep = Math.floor(Math.random() * clickProps.steps.length)
      clickProps.setCurrentStep(nextStep)
    }
  }}
>
  {/* ... */}
</TourProvider>
```

**Comment by jakebanks** [comment]
> Hi @jakebanks, thanks for open the _Issue_ and sorry for the late reply. The new prop [keyboardHandler](https://github.com/elrumordelaluz/reactour/tree/main/packages/tour#keyboardhandler-keyboardhandler) should be now available in `@reactour/tour@3.3.0`.
> 
> You could use it this way:
> 
> ```js
> <TourProvider
>   steps={steps}
>   disableInteraction
>   keyboardHandler={(e, clickProps) => {
>     if (e.key === 'ArrowRight') {
>       clickProps.setCurrentStep(
>         Math.min(clickProps.currentStep + 1, clickProps.steps.length - 1)
>       )
>     }
>     if (e.key === 'ArrowLeft') {
>       clickProps.setCurrentStep(Math.max(clickProps.currentStep - 1, 0))
>     }
>     if (e.key === 'Escape') {
>       const nextStep = Math.floor(Math.random() * clickProps.steps.length)
>       clickProps.setCurrentStep(nextStep)
>     }
>   }}
> >
>   {/* ... */}
> </TourProvider>
> ```

Thanks so much @elrumordelaluz :)
